setup/helpers/perf_helper.rb

```diff
@@ -8,6 +8,10 @@ module PerfHelper
   BASE_URL = 'http://builds.puppetlabs.lan'
   R10K_DIR = '/opt/puppetlabs/scratch/perf_testing/r10k'
   R10K_CONFIG_PATH = "#{R10K_DIR}/r10k.yaml"
+  TEST_FILES_URL = "http://int-resources.ops.puppetlabs.net/QE%20Shared%20Resources/gatling_test_keys".freeze
+  TEST_SSH_FILES = ["id_rsa", "id_rsa.pub", "config", "known_hosts"].freeze
```

Comment:
I don't think we need the public key for anything... we should never be using or storing both the private and public key in the same place.  Having the public key at all here doesn't make sense to me.

Reply:
The change to the known_hosts file and the config file both seem to be trying to solve the same problem. Can we just use the config file and get rid of the known_hosts? Or is the config for SSH directly and known_hosts only for git clone?

Reply:
I wasn't sure if the public key was actually needed; it was included in r10_deploy in the "Set up SSH key for github access" step but it looked like that code may not have ever actually run. I'll remove it.

Reply:
I was thinking the config file was for SSH and known_hosts for git clone. I wanted to try and maintain the existing setup, just with files retrieved from outside the gplt codebase.

Reply:
The private key was the one we want to remove. Is that work still planned?

Reply:
Yes, there was an issue merging that branch so I've created a new branch and PR: https://github.com/puppetlabs/gatling-puppet-load-test/pull/238